terraform/modules/sqs_consumer_task/compute.tf

```diff
@@ -0,0 +1,318 @@
+locals {
+  ecs_service_name = var.namespace
+}
+
+resource "aws_ecs_service" "default" {
+  name                   = local.ecs_service_name
+  cluster                = data.aws_ecs_cluster.default.id
+  task_definition        = aws_ecs_task_definition.consumer.arn
+  desired_count          = 0
+  launch_type            = "FARGATE"
+  enable_execute_command = true
+
+  network_configuration {
+    security_groups  = compact(var.security_group_ids)
+    subnets          = compact(var.subnet_ids)
+    assign_public_ip = false
+  }
+
+  lifecycle {
+    ignore_changes = [desired_count]
+  }
+
+  depends_on = [
+    aws_iam_role.execution,
+    aws_iam_role_policy.execution,
+  ]
+}
+
+module "consumer_task_definition" {
```

Comment:
Should this have a [`container_depends_on`](https://github.com/cloudposse/terraform-aws-ecs-container-definition#input_container_depends_on) for the datadog container?

Reply:
Good call – this PR auto-merged already, but I'll open a new one with this change.